### PR TITLE
Add changelog and the updated pull request template

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

Related issue [#1198](https://github.com/ballerina-platform/ballerina-standard-library/issues/1198)

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Change Log
+This file contains all the notable changes done to the Ballerina Regex package through the releases.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.6.3] - 2021-01-22
```
Comment (shafreenAnfar) [inline changelog.md:7]: I think pre-release tag is missing.

File: .github/workflows/build-master.yml [hunk 2]
```diff
@@ -25,6 +25,8 @@ jobs:
         env:
           packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
           packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
+          username = System.getenv("publishUser")
+          password = System.getenv("publishPAT")
```
Comment (niveathika) [inline .github/workflows/build-master.yml:29]:
```suggestion
          publishUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
          publishPAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
```

**Review by niveathika — CHANGES_REQUESTED** [review]

File: changelog.md (hunk 1 above)
Comment (kalaiyarasiganeshalingam) [reply changelog.md]: It doesn't have a pre-release tag as we don't do any changes after adding APIs in the release version [0.6.0](https://github.com/ballerina-platform/module-ballerina-regex/tree/release-0.6.0).

File: changelog.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Change Log
+This file contains all the notable changes done to the Ballerina Regex package through the releases.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.6.0] - 2021-01-22
```
Comment (shafreenAnfar) [inline changelog.md:7]: I think we need the pre-release tag.

**Review by shafreenAnfar — APPROVED** [review]

**Review by niveathika — APPROVED** [review]

**Pull request merged**